modules/installation-dns-bare-metal.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-dns-bare-metal-{context}"]
+= Bare metal DNS requirements
+
+The following DNS records are required for an {product-title} cluster. In each
+record, `<cluster_name>` is the cluster name and `<base_domain>` is the cluster
+base domain that you specify in the `install-config.yaml` file.
+
+.Required DNS records
+[cols="1a,3a,5a",options="header"]
+|===
+
+|Component
+|Record
+|Description
+
+.2+a|Kubernetes API
+|`api.<cluster_name>.<base_domain>`
+|This DNS record must point to the load balancer
+for the control plane machines. This record must be resolvable by both clients
+external to the cluster and from all the nodes within the cluster.
+
+|`api-int.<cluster_name>.<base_domain>`
+|This DNS record must point to the load balancer
+for the control plane machines. This record must be resolvable from all the
+nodes within the cluster.
+
+|Routes
+|`*.apps.<cluster_name>.<base_domain>`
+|A wildcard DNS record that points to the load balancer that targets the
+machines that  run the ingress router pods, which are the worker nodes by
+default. This record must be resolvable by both clients external to the cluster
+and from all the nodes within the cluster.
+
+.2+|etcd
+|`etcd-<index>.<cluster_name>.<base_domain>`
+|{product-title} requires DNS records for each etcd instance to point to the
+control plane machines that host the instances. The etcd instances are
+differentiated by `<index>` values, which start with `0` and end with `n-1`,
+where `n` is the number of control plane machines in the cluster.  The DNS
+record must resolve to an unicast IPV4 address for the control plane machine,
+and the records must be resolvable from all the nodes in the cluster.
+
+|`etcd-server-ssl._tcp.<cluster_name>.<base_domain>`
+|For each control plane machine, {product-title} also requires a SRV DNS
+record for etcd server on that machine with priority `0`, weight `10` and
+port `2380`. A three control plane machine cluster uses the following records:
```

Comment:
Possibly reword this for clarity/translation:

A machine cluster with three control planes uses the following records:

Reply:
Hmm, I think the current version is more appropriate than the one recommended above.

Reply:
Got it. I'm changing it to "A cluster that uses three control plane machines requires..."